src/utils.test.js

```diff
@@ -149,4 +149,45 @@ describe('utils', () => {
       expect(resultDate.getDate() - initialDate.getDate()).toBe(3);
     });
   });
+
+  describe('getStartOfDate function', () => {
+    it('should return a valid date', () => {
+      // Arrange
+      const initialDate = new Date();
+
+      // Act
+      const resultDate = getStartOfDate(initialDate);
+
+      // Assert
+      expect(resultDate.toISOString()).toBeDefined();
```

Comment:
You can add an assert here to verify if the result date has 0 in hour, minute, second, and ms.

Reply:
Oh, you already did it: https://github.com/FromDoppler/doppler-webapp/pull/542/files#diff-38c429a60240c92cfd160bcc46f1a81bR177

Sorry.

Removing the comment.